daemon/src/housekeeping.rs

```diff
@@ -29,29 +29,28 @@ pub async fn rebroadcast_transactions(
     let cfds = load_all_cfds(conn).await?;
 
     for dlc in cfds.iter().filter_map(|cfd| Cfd::pending_open_dlc(cfd)) {
-        let txid = wallet.try_broadcast_transaction(dlc.lock.0.clone()).await?;
-
+        let txid = try_continue!(wallet.try_broadcast_transaction(dlc.lock.0.clone()).await);
         tracing::info!("Lock transaction published with txid {}", txid);
     }
 
     for cfd in cfds.iter().filter(|cfd| Cfd::is_must_refund(cfd)) {
-        let signed_refund_tx = cfd.refund_tx()?;
-        let txid = wallet.try_broadcast_transaction(signed_refund_tx).await?;
+        let signed_refund_tx = try_continue!(cfd.refund_tx());
```

Comment:
These we should already handle via `filter_map`.

Reply:
ah, yeah, we should change those to be options like for `cfd()` and `attestation` - I can change that as well